### PR TITLE
Write images and aliases when writing content

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: src/main/java/org/atlasapi/input/DescribedModelTransformer.java [hunk 1]
```diff
@@ -53,6 +57,30 @@ private T setDescriptionFields(T result, F inputContent) {
         return result;
     }
 
+    private Iterable<org.atlasapi.media.entity.Image> transformImages(Set<Image> images) {
+        Set<org.atlasapi.media.entity.Image> resultImages = new HashSet<>();
+        if (images == null) {
+            return resultImages;
+        }
+        for (Image image : images) {
```
Comment (tfm) [inline src/main/java/org/atlasapi/input/DescribedModelTransformer.java]: See method below; you can use a `transform` method

File: src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java [hunk 2]
```diff
@@ -44,6 +50,23 @@ private void setEquivalents(T output, F simple) {
         }
     }
 
+    private Iterable<org.atlasapi.media.entity.Alias> transformV4Aliases(Collection<Alias> v4Aliases) {
+        Set<org.atlasapi.media.entity.Alias> aliases = new HashSet<>();
+        if (v4Aliases == null) {
+            return aliases;
+        }
+        for (Alias v4Alias : v4Aliases) {
```
Comment (tfm) [inline src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java]: Same here

File: src/main/java/org/atlasapi/input/DescribedModelTransformer.java (hunk 1 above)
Comment (wwajerowicz) [inline src/main/java/org/atlasapi/input/DescribedModelTransformer.java]: Done

File: src/main/java/org/atlasapi/input/DescribedModelTransformer.java [hunk 3]
```diff
@@ -53,6 +57,26 @@ private T setDescriptionFields(T result, F inputContent) {
         return result;
     }
 
+    private Iterable<org.atlasapi.media.entity.Image> transformImages(Set<Image> images) {
+        if (images == null) {
+            return null;
```
Comment (mbst-jamie) [inline src/main/java/org/atlasapi/input/DescribedModelTransformer.java]: return ImmutableList.of()

File: src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java [hunk 4]
```diff
@@ -33,6 +38,9 @@ private T setIdentifiedFields(T output, F simple) {
         output.setCanonicalUri(simple.getUri());
         output.setCurie(simple.getCurie());
         setEquivalents(output, simple);
+        output.setAliases(
+                transformV4Aliases(simple.getV4Aliases())
```
Comment (mbst-jamie) [inline src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java]: No need for the multi lining here

File: src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java [hunk 5]
```diff
@@ -44,6 +52,21 @@ private void setEquivalents(T output, F simple) {
         }
     }
 
+    private Iterable<org.atlasapi.media.entity.Alias> transformV4Aliases(Collection<Alias> v4Aliases) {
+        if (v4Aliases == null) {
+            return null;
```
Comment (mbst-jamie) [inline src/main/java/org/atlasapi/input/IdentifiedModelTransformer.java]: same as above

**Pull request merged**